controllers/flux/configmap.go

```diff
@@ -25,16 +25,31 @@ import (
 	api "flux-framework/flux-operator/api/v1alpha1"
 )
 
+var BrokerConfigTemplate = `
```

Comment:
We should check with others if this should be handled internally, or in the config/samples as it is now. I'm thinking internal would be ok (but we should double check). I think if we add it here, we should likely remove it from the config samples and the apis and re-generate the manifests too? The drawback of this approach is that we can't generalize the functions (as we did previously) and have to have the logic to check for the config name and do custom logic.

Reply:
Thats a good point, my thoughts are that users will eventually just want to submit something like a flux job spec and have the operator use that to generate all of the required configuration. For now, I think the size is a good proxy for that while we get things started. 

I think the hostfile will certainly have to be generated internally since there's no way for the user to know the hostnames or ip addresses.